src/ServiceBus.AttachmentPlugin/MessageExtensions.cs

```diff
@@ -25,5 +25,25 @@ public static async Task<Message> DownloadAzureStorageAttachment(this Message me
             var plugin = new AzureStorageAttachment(configuration);
             return await plugin.AfterMessageReceive(message).ConfigureAwait(false);
         }
+
+        /// <summary>Download attachment from Azure Storage blob without registering plugin, using default message property to identify SAS URI.</summary>
+        /// <param name="message"><see cref="Message"/></param>
+        /// <returns><see cref="Message"/> with body downloaded from Azure Storage blob.</returns>
+        public static async Task<Message> DownloadAzureStorageAttachment(this Message message)
```

Comment:
Redundant, see comment for the overload.

Reply:
Fixed, thanks!